static/css/components/work.less

```
@@ -370,5 +370,10 @@ div.editionTools {
   font-weight: normal;
   color: @link-blue;
 }
+@media only screen and (max-width: 960px){
```

```suggestion
@media only screen and (max-width: @width-breakpoint-desktop){
```
We use Less.js variables for our breakpoints.  This way, if we have to change the breakpoints in the future, we only need to make the change in a single place.